server/intelligence-service/src/config.py

```diff
@@ -1,5 +1,17 @@
+from dotenv import load_dotenv
+import os
+
+load_dotenv()
+
 class Settings:
     APP_NAME: str = "Intelligence Service"
+    # Non-Azure OpenAI Configuration
+    OPENAI_API_KEY = os.getenv("OPENAI_API_KEY")
```

Comment:
Maybe something like this?
```suggestion
    IS_OPENAI_AVAILABLE = bool(os.getenv("OPENAI_API_KEY"))
    IS_AZURE_OPENAI_AVAILABLE = bool(os.getenv("AZURE_OPENAI_API_KEY")) and bool(os.getenv("AZURE_OPENAI_ENDPOINT")) and bool(os.getenv("AZURE_OPENAI_API_VERSION"))
```